### PR TITLE
Modernize `CondTools/SiPixel`

### PR DESCRIPTION
**Pull request by mmusich**
#### PR description:

Folllow-up to https://github.com/cms-sw/cmssw/pull/35421:
   * moved plugin modules from `test` to `plugins` folder;
   * migrate away from deprecated `edm::EDAnalyzer` ;
   * migrate to `esConsumes`;
   * clean-up of some unused methods;
   * removal of `SealModules` file
   * remove `cout`s in favour of `MessageLogger`

Still possible improvements:
   * merge of header files within the implementation file;
   * introduce more unit test to cover the plugins;

#### PR validation:

It compiles.

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

N/A

### CODE REVIEW

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by tvami** [comment]
I'd add to the todo list of the upcoming PR that unit tests should be added to ensure that these are running fine

**Comment by tvami** [comment]
FYI @SanjanaSekhar

**Comment by mmusich** [comment]
> I found compilation error when building:

actually not sure why this happened. Locally it compiles.

**Comment by mmusich** [comment]
I can reproduce with:
```
cmsrel CMSSW_12_1_X_2021-09-30-1100/
cd CMSSW_12_1_X_2021-09-30-1100/src/
cmsenv
git cms-merge-topic 35495
scramv1 b -j 20
```
how does one get rid of the poisoned files?

**Comment by smuzaffar** [comment]
@mmusich , one of the file still use the deleted header.
./CondTools/SiPixel/plugins/SiPixelVCalReader.h:#include "CondTools/SiPixel/test/SiPixelVCalDB.h"

**Comment by mmusich** [comment]
nevermind found that out myself

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by ggovi** [comment]
please test

**Comment by mmusich** [comment]
sorry @ggovi I found few other mistakes and fix them + adding more tests.

**Comment by mmusich** [comment]
if you don't mind I will trigger the tests once I am confident everything is fine.
I would appreciate if @SanjanaSekhar would take care of the other plugins.

**Comment by mmusich** [comment]
@cmsbuild, please test.
OK I convinced myself the more used ones are OK.

**Comment by tvami** [comment]
+alca
 * comparison shows diff only in msg logger
 * new introduced unit tests pass: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-54096c/19318/unitTests/src/CondTools/SiPixel/test/testPixelDBs/testing.log
 (tho there are some unexpected printouts in TESTING SiPixelLorentzAngle DB codes, but this is beyond the scope of this PR)

File: CondTools/SiPixel/plugins/PixelDCSObjectReader.h [hunk 1]
```diff
@@ -12,30 +12,31 @@
 
 #include "TTree.h"
 
-#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
 #include "FWCore/Framework/interface/ESHandle.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/PixelDCSObjectReader.h:16]: This is not needed any more

File: CondTools/SiPixel/plugins/PixelPopConCalibChecker.cc [hunk 2]
```diff
@@ -6,33 +6,24 @@
 // Aug 2008
 
 #include <iostream>
-
-#include "CondTools/SiPixel/test/PixelPopConCalibChecker.h"
-
+#include "CondTools/SiPixel/plugins/PixelPopConCalibChecker.h"
 #include "FWCore/Framework/interface/EventSetup.h"
 #include "FWCore/Framework/interface/ESHandle.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/PixelPopConCalibChecker.cc:11]: not needed

File: CondTools/SiPixel/plugins/PixelPopConCalibChecker.cc [hunk 3]
```diff
@@ -6,33 +6,24 @@
 // Aug 2008
 
 #include <iostream>
-
-#include "CondTools/SiPixel/test/PixelPopConCalibChecker.h"
-
+#include "CondTools/SiPixel/plugins/PixelPopConCalibChecker.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/PixelPopConCalibChecker.cc:9]:
```suggestion
#include "PixelPopConCalibChecker.h"
```

File: CondTools/SiPixel/plugins/SiPixelBadModuleByHandBuilder.h [hunk 4]
```diff
@@ -1,6 +1,7 @@
 #ifndef SiPixelBadModuleByHandBuilder_H
 #define SiPixelBadModuleByHandBuilder_H
 
+#include "FWCore/Framework/interface/MakerMacros.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "FWCore/Utilities/interface/Exception.h"
 #include "FWCore/Framework/interface/ESHandle.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelBadModuleByHandBuilder.h:7]: not needed

File: CondTools/SiPixel/plugins/SiPixelBadModuleReader.cc [hunk 5]
```diff
@@ -1,23 +1,13 @@
-#include "CondFormats/SiPixelObjects/interface/SiPixelQuality.h"
-#include "CondFormats/DataRecord/interface/SiPixelQualityFromDbRcd.h"
-#include "CondFormats/DataRecord/interface/SiPixelQualityRcd.h"
-#include "CondFormats/DataRecord/interface/SiPixelFedCablingMapRcd.h"
-#include "CondFormats/SiPixelObjects/interface/SiPixelFedCablingMap.h"
-#include "CondTools/SiPixel/test/SiPixelBadModuleReader.h"
+#include "CondTools/SiPixel/plugins/SiPixelBadModuleReader.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelBadModuleReader.cc:1]:
```suggestion
#include "SiPixelBadModuleReader.h"
```

File: CondTools/SiPixel/plugins/SiPixelBadModuleReader.h [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+// user include files
+#include "CondFormats/DataRecord/interface/SiPixelFedCablingMapRcd.h"
+#include "CondFormats/DataRecord/interface/SiPixelQualityFromDbRcd.h"
+#include "CondFormats/DataRecord/interface/SiPixelQualityRcd.h"
+#include "CondFormats/SiPixelObjects/interface/SiPixelFedCabling.h"
+#include "CondFormats/SiPixelObjects/interface/SiPixelFedCablingMap.h"
+#include "CondFormats/SiPixelObjects/interface/SiPixelQuality.h"
+#include "DataFormats/DetId/interface/DetId.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "FWCore/Framework/interface/ESHandle.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelBadModuleReader.h:15]: not needed

File: CondTools/SiPixel/plugins/SiPixelBadModuleReader.cc [hunk 7]
```diff
@@ -1,23 +1,13 @@
...
 #include "DataFormats/DetId/interface/DetId.h"
-#include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
-#include "CondFormats/SiPixelObjects/interface/SiPixelFedCabling.h"
 #include "CondFormats/SiPixelObjects/interface/PixelROC.h"
 #include "CondFormats/SiPixelObjects/interface/PixelFEDCabling.h"
 #include "Geometry/CommonDetUnit/interface/PixelGeomDetUnit.h"
-#include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
 #include "FWCore/ServiceRegistry/interface/Service.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelBadModuleReader.cc:6]: Already in the header

File: CondTools/SiPixel/plugins/SiPixelCondObjAllPayloadsReader.cc [hunk 8]
```diff
@@ -1,6 +1,6 @@
 #include <memory>
 
-#include "CondTools/SiPixel/test/SiPixelCondObjAllPayloadsReader.h"
+#include "CondTools/SiPixel/plugins/SiPixelCondObjAllPayloadsReader.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelCondObjAllPayloadsReader.cc:3]:
```suggestion
#include "SiPixelCondObjAllPayloadsReader.h"
```

File: CondTools/SiPixel/plugins/SiPixelCondObjForHLTBuilder.cc [hunk 9]
```diff
@@ -1,13 +1,10 @@
 #include <memory>
 #include <iostream>
-#include "CondTools/SiPixel/test/SiPixelCondObjForHLTBuilder.h"
+#include "CondTools/SiPixel/plugins/SiPixelCondObjForHLTBuilder.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelCondObjForHLTBuilder.cc:3]:
```suggestion
#include "SiPixelCondObjForHLTBuilder.h"
```

File: CondTools/SiPixel/plugins/SiPixelCondObjForHLTBuilder.h [hunk 10]
```diff
@@ -18,29 +18,34 @@
...
 //
 //
+
+#include "FWCore/Framework/interface/MakerMacros.h"
 #include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/Framework/interface/EventSetup.h"
 #include "FWCore/Framework/interface/ESHandle.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelCondObjForHLTBuilder.h:27]: not needed

File: CondTools/SiPixel/plugins/SiPixelCondObjForHLTBuilder.h [hunk 11]
```diff
@@ -18,29 +18,34 @@
...
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 //#include "CondFormats/SiPixelObjects/interface/SiPixelGainCalibrationForHLT.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelCondObjForHLTBuilder.h:29]: Remove unneeded commented out lines

File: CondTools/SiPixel/plugins/SiPixelCondObjForHLTReader.cc [hunk 12]
```diff
@@ -1,6 +1,6 @@
 #include <memory>
 
-#include "CondTools/SiPixel/test/SiPixelCondObjForHLTReader.h"
+#include "CondTools/SiPixel/plugins/SiPixelCondObjForHLTReader.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelCondObjForHLTReader.cc:3]:
```suggestion
#include "SiPixelCondObjForHLTReader.h"
```

File: CondTools/SiPixel/plugins/SiPixelCondObjOfflineBuilder.cc [hunk 13]
```diff
@@ -1,6 +1,6 @@
 #include <memory>
 #include <iostream>
-#include "CondTools/SiPixel/test/SiPixelCondObjOfflineBuilder.h"
+#include "CondTools/SiPixel/plugins/SiPixelCondObjOfflineBuilder.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelCondObjOfflineBuilder.cc:3]:
```suggestion
#include "SiPixelCondObjOfflineBuilder.h"
```

File: CondTools/SiPixel/plugins/SiPixelCondObjOfflineReader.cc [hunk 14]
```diff
@@ -1,6 +1,6 @@
 #include <memory>
 
-#include "CondTools/SiPixel/test/SiPixelCondObjOfflineReader.h"
+#include "CondTools/SiPixel/plugins/SiPixelCondObjOfflineReader.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelCondObjOfflineReader.cc:3]:
```suggestion
#include "SiPixelCondObjOfflineReader.h"
```

File: CondTools/SiPixel/plugins/SiPixelCondObjReader.cc [hunk 15]
```diff
@@ -1,6 +1,6 @@
 #include <memory>
 
-#include "CondTools/SiPixel/test/SiPixelCondObjReader.h"
+#include "CondTools/SiPixel/plugins/SiPixelCondObjReader.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelCondObjReader.cc:3]:
```suggestion
#include "SiPixelCondObjReader.h"
```

File: CondTools/SiPixel/plugins/SiPixelDynamicInefficiencyDB.cc [hunk 16]
```diff
@@ -4,17 +4,11 @@
 #include <fstream>
 #include <limits>
 #include "CondCore/DBOutputService/interface/PoolDBOutputService.h"
-#include "CondTools/SiPixel/test/SiPixelDynamicInefficiencyDB.h"
+#include "CondTools/SiPixel/plugins/SiPixelDynamicInefficiencyDB.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelDynamicInefficiencyDB.cc:7]:
```suggestion
#include "SiPixelDynamicInefficiencyDB.h"
```

File: CondTools/SiPixel/plugins/SiPixelDynamicInefficiencyDB.cc [hunk 17]
```diff
@@ -4,17 +4,11 @@
...
 #include "Geometry/CommonDetUnit/interface/PixelGeomDetUnit.h"
 #include "CondFormats/SiPixelObjects/interface/SiPixelDynamicInefficiency.h"
 #include "FWCore/Framework/interface/ESHandle.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelDynamicInefficiencyDB.cc:10]: not needed

File: CondTools/SiPixel/plugins/SiPixelDynamicInefficiencyDB.h [hunk 18]
```diff
@@ -3,30 +3,27 @@
 
 #include <map>
 
-#include "FWCore/Framework/interface/EDAnalyzer.h"
-#include "FWCore/Framework/interface/Event.h"
 #include "DataFormats/Common/interface/Handle.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelDynamicInefficiencyDB.h:6]: not used

File: CondTools/SiPixel/plugins/SiPixelDynamicInefficiencyReader.cc [hunk 19]
```diff
@@ -1,19 +1,12 @@
-#include "CondFormats/SiPixelObjects/interface/SiPixelDynamicInefficiency.h"
-#include "CondFormats/DataRecord/interface/SiPixelDynamicInefficiencyRcd.h"
 #include "Geometry/CommonDetUnit/interface/PixelGeomDetUnit.h"
-#include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
-#include "CondTools/SiPixel/test/SiPixelDynamicInefficiencyReader.h"
+#include "CondTools/SiPixel/plugins/SiPixelDynamicInefficiencyReader.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelDynamicInefficiencyReader.cc:2]:
```suggestion
#include "SiPixelDynamicInefficiencyReader.h"
```

File: CondTools/SiPixel/plugins/SiPixelDynamicInefficiencyReader.h [hunk 20]
```diff
@@ -5,27 +5,37 @@
 //#include <memory>
 
 // user include files
+#include "FWCore/Framework/interface/MakerMacros.h"
 #include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/Framework/interface/EventSetup.h"
 #include "FWCore/Framework/interface/ESHandle.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelDynamicInefficiencyReader.h:13]: not needed

File: CondTools/SiPixel/plugins/SiPixelGainCalibrationReadDQMFile.cc [hunk 21]
```diff
@@ -51,7 +51,7 @@
 #include "TList.h"
 #include "TTree.h"
 
-#include "CondTools/SiPixel/test/SiPixelGainCalibrationReadDQMFile.h"
+#include "CondTools/SiPixel/plugins/SiPixelGainCalibrationReadDQMFile.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelGainCalibrationReadDQMFile.cc:54]:
```suggestion
#include "SiPixelGainCalibrationReadDQMFile.h"
```

File: CondTools/SiPixel/plugins/SiPixelLorentzAngleDB.cc [hunk 22]
```diff
@@ -3,17 +3,11 @@
 #include <iostream>
 #include <fstream>
 #include "CondCore/DBOutputService/interface/PoolDBOutputService.h"
-#include "CondTools/SiPixel/test/SiPixelLorentzAngleDB.h"
+#include "CondTools/SiPixel/plugins/SiPixelLorentzAngleDB.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelLorentzAngleDB.cc:6]:
```suggestion
#include "SiPixelLorentzAngleDB.h"
```

File: CondTools/SiPixel/plugins/SiPixelLorentzAngleReader.cc [hunk 23]
```diff
@@ -1,9 +1,6 @@
-#include "CondFormats/SiPixelObjects/interface/SiPixelLorentzAngle.h"
-#include "CondFormats/DataRecord/interface/SiPixelLorentzAngleRcd.h"
-#include "CondFormats/DataRecord/interface/SiPixelLorentzAngleSimRcd.h"
 #include "Geometry/CommonDetUnit/interface/PixelGeomDetUnit.h"
 #include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
-#include "CondTools/SiPixel/test/SiPixelLorentzAngleReader.h"
+#include "CondTools/SiPixel/plugins/SiPixelLorentzAngleReader.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelLorentzAngleReader.cc:3]:
```suggestion
#include "SiPixelLorentzAngleReader.h"
```

File: CondTools/SiPixel/plugins/SiPixelLorentzAngleDB.h [hunk 24]
```diff
@@ -3,12 +3,17 @@
 
 #include <map>
 
-#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
 #include "FWCore/Framework/interface/Event.h"
 #include "DataFormats/Common/interface/Handle.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelLorentzAngleDB.h:9]: not needed?

File: CondTools/SiPixel/plugins/SiPixelLorentzAngleReader.h [hunk 25]
```diff
@@ -5,14 +5,18 @@
 //#include <memory>
 
 // user include files
+#include "FWCore/Framework/interface/MakerMacros.h"
 #include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/Framework/interface/EventSetup.h"
 #include "FWCore/Framework/interface/ESHandle.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelLorentzAngleReader.h:13]: not needed

File: CondTools/SiPixel/plugins/SiPixelPerformanceSummaryReader.cc [hunk 26]
```diff
@@ -1,9 +1,5 @@
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
-
-#include "CondFormats/SiPixelObjects/interface/SiPixelPerformanceSummary.h"
-#include "CondFormats/DataRecord/interface/SiPixelPerformanceSummaryRcd.h"
-
-#include "CondTools/SiPixel/test/SiPixelPerformanceSummaryReader.h"
+#include "CondTools/SiPixel/plugins/SiPixelPerformanceSummaryReader.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelPerformanceSummaryReader.cc:2]:
```suggestion
#include "SiPixelPerformanceSummaryReader.h"
```

File: CondTools/SiPixel/plugins/SiPixelPerformanceSummaryReader.h [hunk 27]
```diff
@@ -1,23 +1,27 @@
 #ifndef SiPixelPerformanceSummaryReader_H
 #define SiPixelPerformanceSummaryReader_H
 
+#include "FWCore/Framework/interface/MakerMacros.h"
 #include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/Framework/interface/EventSetup.h"
 #include "FWCore/Framework/interface/ESHandle.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelPerformanceSummaryReader.h:9]: not needed?

File: CondTools/SiPixel/plugins/SiPixelVCalDB.cc [hunk 28]
```diff
@@ -1,31 +1,27 @@
-#include "CondTools/SiPixel/test/SiPixelVCalDB.h"
+#include "CondTools/SiPixel/plugins/SiPixelVCalDB.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelVCalDB.cc:1]:
```suggestion
#include "SiPixelVCalDB.h"
```

File: CondTools/SiPixel/plugins/SiPixelVCalDB.h [hunk 29]
```diff
@@ -24,16 +25,16 @@
 #include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
 //#include "CondTools/SiPixel/test/SiPixelVCalPixelId.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelVCalDB.h:26]: remove unneeded commented out line

File: CondTools/SiPixel/plugins/SiPixelVCalReader.cc [hunk 30]
```diff
@@ -1,34 +1,36 @@
-#include "CondTools/SiPixel/test/SiPixelVCalReader.h"
+#include "CondTools/SiPixel/plugins/SiPixelVCalReader.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelVCalReader.cc:1]:
```suggestion
#include "SiPixelVCalReader.h"
```

File: CondTools/SiPixel/plugins/SiPixelVCalReader.h [hunk 31]
```diff
@@ -1,19 +1,20 @@
 #ifndef SiPixelVCalReader_H
 #define SiPixelVCalReader_H
 
-#include <stdio.h>
-#include <sys/time.h>
-#include <iomanip>  // std::setw
-#include <iostream>
-#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
 #include "FWCore/Framework/interface/ESHandle.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelVCalReader.h:5]: not needed

File: CondTools/SiPixel/plugins/SiPixelVCalReader.h [hunk 32]
```diff
@@ -31,20 +32,24 @@
 #include "CondFormats/DataRecord/interface/SiPixelVCalRcd.h"
 #include "CondFormats/DataRecord/interface/SiPixelVCalSimRcd.h"
 #include "CondFormats/SiPixelObjects/interface/SiPixelVCal.h"
-#include "CondTools/SiPixel/test/SiPixelVCalDB.h"
+#include "CondTools/SiPixel/plugins/SiPixelVCalDB.h"
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelVCalReader.h:35]:
```suggestion
#include "SiPixelVCalDB.h"
```

**Review by perrotta — COMMENTED** [review]
The PR looks correct, and it could even be merged as such.
However, while scrolling through it I found a few possible further cleanings, linked to what you applied here, that could also get implemented. I let you decide whether yu want to implement them here (which is probably the fastest thing to do, since they are already all listed), or in a forthcoming cleaning PR. 
All those suggested fixes are not necessary at all, and we could even go ahead without them: just let me know if you can't deal with them. But since they can be easily applied, why not?

File: CondTools/SiPixel/plugins/SiPixelBadModuleReader.h [hunk 33]
```diff
@@ -0,0 +1,57 @@
+#ifndef SiPixelBadModuleReader_H
```
Comment (perrotta) [inline CondTools/SiPixel/plugins/SiPixelBadModuleReader.h:1]:
```suggestion
#ifndef CondTools_SiPixel_SiPixelBadModuleReader_H
#define CondTools_SiPixel_SiPixelBadModuleReader_H
```

**Comment by mmusich** [comment]
Thanks @perrotta I'll deal with them later today,  this PR is not urgent anyway

**Comment by mmusich** [comment]
@perrotta all review comments have been addressed. 

@tvami's https://github.com/cms-sw/cmssw/pull/35495#issuecomment-932661159 about  unexpected printouts in TESTING SiPixelLorentzAngle DB codes is addressed at e1de205

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by tvami** [comment]
+db

**Comment by perrotta** [comment]
+1

**Pull request merged**